configure

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+#
+# Copyright(c) 2012-2019 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+MODULE_FILE=test_mod.c
+OBJ_MOD=test_mod.o
+KERN_VER=`uname -r`
+PWD=`pwd`
+NPROC=`nproc`
+DEFINE_FILE=modules/generated_defines.h
+
+
+add_define() {
+   echo -n "#define " >> $DEFINE_FILE
+   for arg in "$@"; do
+       echo -e "$arg" >> $DEFINE_FILE
+   done
+}
+
+add_function() {
```

Comment:
This function isn't used - maybe remove it?

Reply:
It will be used in future commits - when we will be getting rid of current linux_kernel_version.h